galsim/interpolant.py

```diff
@@ -30,16 +39,19 @@
 from .integ import int1d
 from .bessel import si
 
+
```

Comment:
Please remove the whitespace changes, which are unconnected to your new code.

Reply:
I guess it is due Black formatting that I have activated in VSC. Ok I will remove them.

Reply:
In fact Black had changed several formatting features. Try to "deBlack" ...

Reply:
Hope that it's ok now. I have verify that the tests are passed.